pkg/server/container_attach.go

```diff
@@ -38,7 +38,7 @@ func (c *criContainerdService) Attach(ctx context.Context, r *runtime.AttachRequ
 
 	cntr, err := c.containerStore.Get(r.GetContainerId())
 	if err != nil {
-		return nil, fmt.Errorf("failed to find container in store: %v", err)
+		return nil, fmt.Errorf("failed to find container %q in store: %v", r.GetContainerId(), err)
```

Comment:
I don't understand.  Why add it at below?
```
func (c *criContainerdService) Attach(ctx context.Context, r *runtime.AttachRequest) (retRes *runtime.AttachResponse, retErr error) {
	glog.V(2).Infof("Attach for %q with tty %v and stdin %v", r.GetContainerId(), r.GetTty(), r.GetStdin())
	defer func() {
		if retErr == nil {
			glog.V(2).Infof("Attach for %q returns URL %q", r.GetContainerId(), retRes.Url)
		}
	}()
```

Reply:
@yanxuean That's log. For log we should include the id, so that we could see what container is throwing this error in log. :)

Any way, I just feel like, on kubelet side, when there is an error, we'll see the id in error 2 or more times in the error message, which is a bit redundant to me. So I kind of prefer not including container id in our error message if caller already knows it.

Reply:
I see. Thanks